File: Lista pt3/Exercicio01Arrays.java
import java.util.Scanner;

public class Exercicio01Arrays {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        String[] times = new String[10];

        for (int i = 0; i < times.length; i++) {

            System.out.print(
                    "Digite o " + (i + 1) + "º time: "
            );

            times[i] = sc.nextLine();
        }

        System.out.println("\nTimes cadastrados:");

        for (int i = 0; i < times.length; i++) {
            System.out.println(times[i]);
        }

        sc.close();
    }
}

File: Lista pt3/Exercicio01Arrays.js
let times = [];

for (let i = 0; i < 10; i++) {
    let time = prompt("Digite o " + (i + 1) + "º time:");

    times.push(time);
}

console.log("Times cadastrados:");

for (let i = 0; i < times.length; i++) {
    console.log(times[i]);
}